### ISSUE TITLE
Problem: Nim version

### ISSUE THREAD

**Issue by fontire** [TurtleP/lovebrew#13]
Unfortunately, I don't see anything "easy" about Nim/Nimble.

I tried the Debian package which told me it was:

```
Nim Compiler Version 0.19.4 [Linux: amd64]
Compiled at 2019-03-05
```

Then I went to the Nim webpage and downloaded the binary packages from there:

```
Nim Compiler Version 1.4.8 [Linux: amd64]
Compiled at 2021-05-25
```

Their install script didn't install nimble, for some reason but in any case, when I run `nimble build` I get:

```
 Verifying dependencies for lovebrew@0.4.0
       Tip: 3 messages have been suppressed, use --verbose to show them.
     Error: Unsatisfied dependency: nim (>= 1.5.0)
```

The official version from the website is 1.4.8, so I'm not sure where I'm supposed to get 1.5.0+.

**Comment by TurtleP** [comment]
Yeah that's a development version, it's fixed in 0.5.0, which uses the latest stable version. My apologies for that. You also shouldn't need to build from source, but this might be related to your previous issue?

**Comment by fontire** [comment]
Right, I'm only building due to the other issue. I think this issue is solved then.
